chart/monitoring/ClusterSelector.vue

```diff
@@ -85,7 +99,7 @@ export default {
       case 'RKE':
```

Comment:
All these words should a) be translations, and b) be keyed off the values the `currentCluster.status.provider` can have (k3s, rke2, etc) instead of an English string.

Reply:
I've pushed new changes that I think both accomplish what you want and simplify the logic. LMK if you want more changes.